HLTrigger/special/src/HLTTrackerHaloFilter.cc

```diff
@@ -58,7 +57,7 @@ bool HLTTrackerHaloFilter::hltFilter(edm::Event& event, const edm::EventSetup& i
 /*
   //Retrieve tracker topology from geometry
   edm::ESHandle<TrackerTopology> tTopoHandle;
-  iSetup.get<IdealGeometryRecord>().get(tTopoHandle);
+  iSetup.get<TrackerTopologyRcd>().get(tTopoHandle);
   const TrackerTopology* const tTopo = tTopoHandle.product();
```

Comment:
Never mind, I missed that the whole body(!) of the filter code is commented out and the filter
returns TRUE always ... uuhhh

OK, fine!